osidb/models.py

```diff
@@ -832,6 +826,19 @@ def _validate_notabug_flaw_affected(self):
                 f"{affected.ps_component} is affected by a flaw solved as NOTABUG.",
             )
 
+    def _validate_flaw_affects_status_resolution(self):
```

Comment:
I see your point and had second thoughts on my implementation, I will remove this validation because I think it will never be reached once we only interact with affects on this matter. Thank you.